src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+import missedArticles from './_data/missed-articles.json';
+import yourArticles from'./_data/your-articles.json';
+
+ReactDOM.render(<App missedArticles={missedArticles} yourArticles={yourArticles}/>,
+    document.getElementById('root'));
```

Comment:
This works, but makes the data immutable since props cannot be modified. When we start using databases and api calls it will be more useful to store data in the Apps state so it is easier to manipulate.